client/src/components/new-request/search.vue

```diff
@@ -464,15 +499,15 @@ export default class NewSearch extends Vue {
   .v-select__selections {
     line-height: 1.5rem !important;
   }
-  /* remove bottom margin from checkboxes */
-  .v-input--checkbox > .v-input__slot {
```

Comment:
This selector was broken. It should not have the > between the first and second classname.

However, once fixed, it didn't do what I wanted anyway, so I just deleted it.